src/framework/sse/core.clj

```diff
@@ -58,6 +59,12 @@
   (let [events-channel (get-in state [:deps :events-channel :channel])]
     (async/put! events-channel message)))
 
+(defn- put->session
```

Comment:
I'm guessing this is needed for the next PR... not called here.

Reply:
It's necessary, but should not be private. Thanks @soulflyer !